man/tss2-tcti-cmd.7.in

```diff
@@ -12,3 +12,20 @@ with the interface and protocol exposed by a sub-process that can receive and tr
 raw TPM2 command and response buffers. The interface exposed by this library is defined
 in the \*(lqTSS System Level API and TPM Command Transmission Interface
 Specification\*(rq specification.
+.SH EXAMPLES
+.sp
+For example, if you wanted to use the tpm2_send(1) command as the sub-process to
+send data to and from the TPM, one could do so like this:
+.nf
+tpm2_getrandom -T "cmd:tpm2_send -s" --hex 4"
```

Comment:
I think the " after --hex 4 should be removed.

Reply:
Ahh yep